Backpack-SwiftUI/FlightLeg/Classes/BPKFlightLeg.swift

```diff
@@ -0,0 +1,177 @@
+import SwiftUI
+import Backpack_Common
+
+struct BPKFlightLeg: View {
+    private let airlineLogoSize: CGFloat = 24.0
+
+    private let departureArrivalTime: String
+    private let nextDayArrival: String?
+    private let flightDescription: AttributedString
+    private let stopsInfo: String
+    private let highlightStopsInfo: Bool
+    private let duration: String
+    private let operatedBy: String?
+    private let warning: String?
+    @State private var carrierLogo: Image?
+
+    // String-based init
+    public init(
+        departureArrivalTime: String,
+        nextDayArrival: String? = "",
```

Comment:
This name is not very clear at describing what's this for, it should rather be named after what's the label intention, rather than the potential content. as in, is it a comment on the view, information badge, etc. if it's exclusively for something about the next day arrival, then it should be a bit clearer on what's expected to be passed here

Reply:
aside: Why receive an empty string when this is optional? it shold be nil instead. if it's really not intended to be optional, then it should not be, but looks like it should